docs/requirements/requirements.md

```diff
@@ -333,6 +342,13 @@ See also R-U-003.
 
 - What other variants could be interesting?
 
+#### R-Conf-006: Configurable reliability
+
+- The system should be configurable regarding its services' reliability.
+- This would be "nice" for testing with chaos testing tools.
```

Comment:
Doesn't the chaos toolkit simulate unreliable services by randomly killing them / doing other stuff?
Why does our system then need to be configurable too regarding service reliability (besides using the chaos toolkit)?

Reply:
or what do you exactly mean with "configurable reliability"?

Reply:
Yeah, merging into the other requirement sounds good